cmd/deploy/endpoints.go

```diff
@@ -180,23 +195,33 @@ func (eg *EndpointGetter) getEndpoints(ctx context.Context, opts *EndpointsOptio
 
 	sanitizedName := format.ResourceK8sMetaString(opts.Name)
 	labelSelector := fmt.Sprintf("%s=%s", model.DeployedByLabel, sanitizedName)
-	iClient, err := eg.K8sClientProvider.GetIngressClient()
-	if err != nil {
-		return nil, err
-	}
-	eps, err := iClient.GetEndpointsBySelector(ctx, opts.Namespace, labelSelector)
-	if err != nil {
-		return nil, err
-	}
 
-	externalEps, err := eg.endpointControl.List(ctx, opts.Namespace, labelSelector)
-	if err != nil {
-		return nil, err
-	}
+	var eps []string
+	var err error
+	if okteto.Context().IsOkteto {
+		eps, err = eg.endpointControl.List(ctx, opts.Namespace, sanitizedName)
+		if err != nil {
+			eps, err = eg.getEndpointsInStandaloneMode(ctx, opts, labelSelector, eg.K8sClientProvider)
+			if err != nil {
+				return nil, err
+			}
 
-	for _, externalEp := range externalEps {
-		for _, ep := range externalEp.Endpoints {
-			eps = append(eps, fmt.Sprintf("%s (external)", ep.Url))
+			externalEps, err := eg.externalResourceControl.List(ctx, opts.Namespace, labelSelector)
```

Comment:
Do we need to query external resources in this case?
If the endpoints query fails, I would show an error and continue. Showing the endpoints is a nice to have, but the command shouldn't fail

Reply:
Yes, that fallback I think is not needed anymore. I did it thinking about creating a new endpoint in the backend and scenarios where okteto chart deployed is not the one containing the new endpoint.. but finally I've reuse `space` query so it shouldn't fail. If the use of the backend endpoint fails I think we should return the error and not fallback. 6ebccf1ff64970e5b09516c5eab7313de63aeb35 WDYT?

Reply:
Makes sense